src/handlers/index.ts

```diff
@@ -114,7 +114,11 @@ neh.addHandler(
   'nixo',
   new SearchEngineHandler(
     'does a NixOS option search',
-    makeHashBasedSearchEngine('https://nixos.org/nixos/options.html', null),
+    makeParamBasedSearchEngine(
+      'https://search.nixos.org/options?channel=unstable',
+      null,
+      'query',
+    ),
```

Comment:
Thanks! I don't actually use NixOS anymore, so if you're not using it either we could just dump these hahaa